### ISSUE TITLE
ExtraAttributes don't fit in the database 

### ISSUE THREAD

**Issue by Silox** [ZeusWPI/FK-enrolment#56]
```
Mysql2::Error: Data too long for column 'value' at row 1: INSERT INTO `extra_attributes` (`spec_id`, `value`, `member_id`, `created_at`, `updated_at`) VALUES (15, '---\n- \'\'\n- \'Sport: voetbal\'\n- \'Sport: zwemmen\'\n- \'Sport: andere\'\n- \'Feest: DJ\'\n- \'Feest: organisatie\'\n- Cultuur\n- Cantus en doop\n- \'Web: design\'\n- \'Web: beheer\'\n- \'SWOP: vertegenwoordiging\'\n- \'SWOP: examenvragen\'\n- \'BeMSA: lokale projecten\'\n- \'BeMSA: internationaal\'\n', 49833, '2015-08-25 10:10:28', '2015-08-25 10:10:28') 
app/controllers/frontend/registration_controller.rb:41:in `general' 
```

**Comment by Silox** [comment]
```
mysql> SHOW COLUMNS FROM extra_attributes;
+------------+--------------+------+-----+---------+----------------+
| Field      | Type         | Null | Key | Default | Extra          |
+------------+--------------+------+-----+---------+----------------+
| id         | int(11)      | NO   | PRI | NULL    | auto_increment |
| member_id  | int(11)      | YES  |     | NULL    |                |
| spec_id    | int(11)      | YES  |     | NULL    |                |
| value      | varchar(255) | YES  |     | NULL    |                |
| created_at | datetime     | YES  |     | NULL    |                |
| updated_at | datetime     | YES  |     | NULL    |                |
+------------+--------------+------+-----+---------+----------------+
6 rows in set (0.00 sec)
```

**Comment by Silox** [comment]
Fixed in 09b398ae8e3779172bffa6306a01ae7e972d38e7
